ci/nightly/pipeline.yml

```diff
@@ -86,6 +86,8 @@ steps:
     label: Build x86_64
     command: bin/ci-builder run stable bin/pyactivate -m ci.test.build x86_64
     timeout_in_minutes: 60
+    # For releases we trigger nightly from the test job directly, no need to build again
+    branches: "!v.v*"
```

Comment:
Are we sure about the second `v` in this regexp and the following ones?  Seems like it will never match.

I think the `.` needs to be escaped here.

Reply:
Sorry, should be `"!v*.*"`, my brain must be slightly fried this morning. No escaping required since this is not a real regex.